point.go

```diff
@@ -338,3 +338,24 @@ func (p Point) ToWKT() string {
 func (p Point) String() string {
 	return fmt.Sprintf("POINT(%g %g)", p[0], p[1])
 }
+
+// Round returns a point which coordinates are rounded
+func (p *Point) Round(decimalPlaces int) *Point {
```

Comment:
I think you can remove `exp := math.Pow(10, float64(decimalPlaces))` calculating and expect that as a parameter.
```
func (p *Point) Round(factor int) *Point {
    exp := float64(factor)
    ...
}

p := geo.Point{-73.23321, 40.84121321}
p.Round(1.0e6)
```
instead of:
```
p.Round(6)
```

WDYT?